weed/storage/volume_vacuum.go

```diff
@@ -484,7 +487,20 @@ func (v *Volume) copyDataBasedOnIndexFile(srcDatName, srcIdxName, dstDatName, da
 	if err != nil {
 		return err
 	}
-
+	dstDatSize, _, err := dstDatBackend.GetStat()
+	if err != nil {
+		return err
+	}
+	if v.nm.ContentSize() > v.nm.DeletedSize() {
+		expectedContentSize := v.nm.ContentSize() - v.nm.DeletedSize()
```

Comment:
when using leveldb, `needleMapMetricFromIndexFile()` function used a bloomfilter to estimate the deleted size. So this expectedContentSize may not be accurate.

Reply:
I looked at the inaccuracy with the size of the deleted bytes; it may be smaller, which will not harm the logic.